protobuf/plan.sh

```diff
@@ -1,11 +1,19 @@
 pkg_name=protobuf
 pkg_origin=core
-pkg_version=2.6.1
+pkg_version=3.6.0
+
+pkg_description="Protocol buffers are a language-neutral, platform-neutral extensible mechanism for serializing structured data."
+pkg_upstream_url="https://developers.google.com/protocol-buffers/"
 pkg_license=('BSD')
-pkg_source=https://github.com/google/${pkg_name}/releases/download/v${pkg_version}/${pkg_name}-${pkg_version}.tar.bz2
-pkg_shasum=ee445612d544d885ae240ffbcbf9267faa9f593b7b101f21d58beceb92661910
-pkg_deps=(core/gcc core/zlib)
+
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_source=https://github.com/google/${pkg_name}/releases/download/v${pkg_version}/${pkg_name}-all-${pkg_version}.tar.gz
+pkg_shasum=1532154addf85080330fdd037949d4653dfce16550df5c70ea0cd212d8aff3af
+
+pkg_deps=(core/gcc
```

Comment:
I hate myself for this. But somewhat of a standard through the other plans (when multiple deps are required) is splitting this fully onto separate lines:

```
pkg_deps=(
  core/depone
  core/deptwo
)
```

Reply:
@predominant No problem at all; force-push incoming.